.mage/mage.make

```diff
@@ -23,16 +23,4 @@ init: $(MAGE)
 	@$(MAGE) init
 	@echo "Run \"./mage -l\" for a list of build targets"
 
-.PHONY: git.pre-commit
-git.pre-commit: $(MAGE) # NOTE: DO NOT CHANGE - will break previously installed git hooks.
```

Comment:
Undo. We can just keep doing this with Make.
Also, as the notes say, this will break existing development environments.